src/javascript/app_2/Stores/Modules/Statement/Helpers/__tests__/format_response.js

```diff
@@ -0,0 +1,32 @@
+import { expect }                       from 'chai';
+import { formatStatementTransaction }   from '../format_response.js';
+
+describe('formatStatementTransaction', () => {
+    const transaction = {
+        action_type     : 'Buy',
+        transaction_time: 123456789,
+        transaction_id  : 1234,
+        payout          : 1000,
+        amount          : 2000,
+        balance_after   : 3000,
+        longcode        : 'test \n test \n test',
+        contract_id     : 1234,
+        app_id          : 1234,
+    };
+
+    const currency = 'USD';
+
```

Comment:
@ahkuma better to add a test in which payout/balance/amount isNaN and check if returns '-' in the result or not

Reply:
got it